PROJECT_flexbe_behaviors/src/PROJECT_flexbe_behaviors/manifests/example_behavior_manifest.py

```diff
@@ -0,0 +1,33 @@
+example_behavior = {
+	'name': 'Example Behavior',
+	'executable': {
+		'package_path': 'PROJECT_flexbe_behaviors.example_behavior_sm',
+		'class': 'ExampleBehaviorSM'
+	},
+
+	'author': 'Philipp Schillinger',
+	'date': 'Fri Aug 21 2015',
+	'tagstring': 'demo',
+	'description': """
+		This is a simple example for a behavior.
+	""",
+
+	# Contained behaviors
+	'contained_behaviors': [
+	],
+
+	# Available parameters
+	'params': [
+		{
+			'type': 'numeric',
+			'name': 'waiting_time',
+			'default': '3',
+			'label': 'Waiting Time',
+			'hint': 'Sets the time to wait after logging.',
+			'additional': {
```

Comment:
See [here](https://github.com/FlexBE/flexbe_app/blob/master/src/io/io_manifestgenerator.js#L59-L73) for the different entries of `additional` based on the `type` field of a parameter.